src/array/list/mutable.rs

```diff
@@ -152,6 +156,75 @@ impl<O: Offset, M: MutableArray> MutableListArray<O, M> {
         }
     }
 
+    /// Expand this array, using elements from the underlying backing array.
+    /// Assumes the expansion begins at the highest previous offset, or zero if
+    /// this [MutableListArray] is currently empty.
+    ///
+    /// Panics if:
+    /// - the new offsets are not in monotonic increasing order.
+    /// - any new offset is not in bounds of the backing array.
+    /// - the passed iterator has no upper bound.
+    pub fn extend_offsets<II>(&mut self, expansion: II)
```

Comment:
```suggestion
    pub(crate) fn extend_offsets<II>(&mut self, expansion: II)
```

let's keep this private for now ^^